iidm/iidm-xml-converter/src/test/resources/V1_1/regulatingStaticVarCompensatorRoundTripRef.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_1" id="svcTestCase" caseDate="2016-06-29T14:54:03.427+02:00" forecastDistance="0" sourceFormat="code">
```

Comment:
To be fixed before the merge?

Reply:
@MioRtia I see you add the `PR: next-release` label. I prefer not to put ready-to-be-merge label to avoid someone to merge it too soon...

Reply:
I updated current version to 1.1 in this PR as decided (we go to 1.1 when merging the first change)